### ISSUE TITLE
Query about model generalization to protein-ligand complexes for drug design

### ISSUE THREAD

**Issue by ORCAaAaA-ui** [lillgroup/HydrationSitePrediction#1]
Hi,

Great work on this method for hydration site analysis.

I'm interested in using your model for analyzing hydration, potentially for water-mediated drug design.

Based on the paper, the model was trained solely on protein data. Could you comment on how well the model is expected to generalize to predict hydration sites and thermodynamics for protein-ligand complexes? My specific interest is in its potential application in this area – can this model provide reliable hydration site predictions and thermodynamic profiling for 'protein-ligand complexes', perhaps comparable in utility to methods like WaterMap, even though it was trained only on apo proteins?

Are the predictions (especially the thermodynamics of waters near the ligand or bridging interactions) considered accurate enough to reliably inform drug design strategies, given the absence of ligands in the training set? Any information on validation or expected performance on complexes for this purpose would be greatly appreciated.

**Comment by hinzflorian** [comment]
Thank you very much for your interest in our work!


Yes, the model was trained solely on protein data, however in holo conformations. In the current form, the model is only designed to accept protein structures as input. So any predictions would be made based on the protein structure only.
In our [work](https://pubs.acs.org/doi/10.1021/acs.jcim.4c02349) in section 4.3.5. we performed a case study predicting hydration sites and thermodynamics based on the protein only, and considered the hydration sites close to the ligand as displaced. 
The accordingly calculated desolvation free energy correlated well with the experimental binding free energy (Figure 9). However, this might likely have been a special case, and we expect the accuracy of positions and thermodynamics of predicted hydration sites to be limited in the general case, specifically spatially close to the ligand.

However we are working right now on a follow-up work, retraining our model on a protein-ligand data set. This work will be available in the near future and specifically address your question.
